packages/workflows/__tests__/cli/list/List.definition.unit.test.ts

```diff
@@ -15,7 +15,7 @@ describe("zos-workflows list definition", () => {
     it ("should not have changed", () => {
         const definition: ICommandDefinition = require("../../../src/cli/list/List.definition").ListDefinition;
         expect(definition).toBeDefined();
-        expect(definition.children.length).toBe(2+1);
+        expect(definition.children.length).toBe(2+1+1);
```

Comment:
Can you just define a constant here for 4 please?

Reply:
Done